freqtrade/constants.py

```diff
@@ -356,6 +356,7 @@
                 "properties": {
                     "method": {"type": "string", "enum": AVAILABLE_PROTECTIONS},
                     "stop_duration": {"type": "number", "minimum": 0.0},
+                    "unlock_at": {"type": "string"},
```

Comment:
As a user - i don't think configuring a "unlock_at" alone will be sufficient.

You might think "unlock at 00:00" - but in reality, that could be 23h or 5min - depending on when the protection triggers
so in reality, i'd probably want to configure a "at 00:00 - but lock for at least x minutes" ... which in reality, complicates this quite some.

Reply:
Hm not me, but that's pretty subjective.

Reply:
This is exactly the point I made previously too. What one person wants from a feature might not make sense for all the other users of that feature, so the preference is to generalise.

Reply:
Well to be fair - that whole feature is pretty subjective.
I've not come across this as a requirement or request before - so i'm not sure if it's actually useful beyond your own usage. 
The fewer it's generalized however, the fewer people will have the chance to use it.


It could effectively be pretty simple to use the `order_filled()` callback in the strategy (you'll get it with closed trades, too), and define whatever logic a user thinks is necessary to determine the end-date and then lock that pair manually.